File: cursoPOOUber/PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar() {
        echo "
            <br>
            Licencia: $this->license <br>
            Driver: {$this->driver->name} <br>
            Número de pasajeros: $this->passenger <br>
            Brand: $this->brand <br>
            Model: $this->model <br>
    
        ";
    }

}
?>

File: cursoPOOUber/Java/Uberx.java
package Java;

class Uberx extends Car{
    String brand;
    String model;

    public Uberx(String license, Driver driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }    

    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Model: " + model + "\nBrand: " + brand);
    }
}
